lib/vdsm/tool/configurators/multipath.py

```diff
@@ -109,3 +105,26 @@ def isconfigured():
 
     sys.stdout.write("multipath requires configuration\n")
     return NO
+
+
+def isconfigured():
+    """
+    Check the multipath daemon configuration.
+    """
+    conf_metadata_ok = _check_mpath_metadata()
+    if conf_metadata_ok == YES:
+        check_result = mpathconf.check_local_config()
+        if check_result.error is not None:
+            # Just warn for now, do not fail the isconfigured check here.
+            sys.stdout.write(
+                f"WARNING: Invalid configuration: {check_result.error}:\n")
```

Comment:
If the check failed because running "multipathd show config local" failed, we log now:

    WARNING: Invalid configuration: error from multpathd...

Instead of:

    WARNING: Cannot check multipath configuration : error from multipathd...